db/projects/omisego/asset-omg-logo-square.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2000 2000" width="2500" height="2500"><circle cx="1000" cy="1000" r="1000" fill="#1a53f0"/><path d="M803.2 1135.44c-127.71 0-231.24 103.83-231.24 231.91s103.54 231.91 231.25 231.91 231.25-103.83 231.25-231.91-103.53-231.91-231.25-231.91zm0 393.67c-89 0-161.21-72.38-161.21-161.68s72.18-161.68 161.21-161.68 161.22 72.38 161.22 161.68c-.15 89.22-72.24 161.57-161.22 161.68zm493.86-888.36c-127.71 0-231.25 103.83-231.25 231.91s103.54 231.91 231.25 231.91 231.25-103.83 231.25-231.91-103.52-231.92-231.24-231.92zm0 393.67c-89 0-161.21-72.38-161.21-161.68s72.15-161.68 161.22-161.68 161.22 72.38 161.22 161.68c-.15 89.26-72.29 161.52-161.22 161.67zM803.2 640.74c-110.09-.07-205 77.69-226.88 185.9S611.63 1043.53 713 1086.5a231.86 231.86 0 0 0 255.82-375.61h65.57v-70.15H803.2zm161.23 231.91c0 89.29-72.18 161.68-161.22 161.68S642 961.94 642 872.65 714.17 711 803.2 711s161.15 72.41 161.31 161.68h-.08z" fill="#fff"/></svg>
```

Comment:
This is not a standard square logo which OmiseGo uses, standard logo is just "go" like they have on every social networks or coinmarketcap
There is right logo: https://www.coinfairvalue.com/static/svg/coins/omisego.svg

Reply:
This logo we can use as logo-alternative maybe

Reply:
The origin logo was changed to asset-omg-logo-alternative.svg. The logo from the url was set as asset-omg-logo-square.svg